deploy/charts/cert-manager/templates/cainjector-config.yaml

```diff
@@ -1,6 +1,7 @@
 {{- if .Values.cainjector.config -}}
-{{- $_ := .Values.cainjector.config.apiVersion | required ".Values.cainjector.config.apiVersion must be set !" -}}
-{{- $_ := .Values.cainjector.config.kind | required ".Values.cainjector.config.kind must be set !" -}}
+{{- $config := .Values.cainjector.config -}}
```

Comment:
Could you add some comments in the values.yaml file about backwards compatibility etc?